src/js/pages/customer.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+import SelectorsMap from '@constants/selectors-map';
+
+const initCustomer = () => {
+  const returnTableMainCheckbox = document.querySelector(SelectorsMap.order.returnFormMainCheckbox) as HTMLInputElement;
```

Comment:
```suggestion
  const returnTableMainCheckbox = document.querySelector<HTMLInputElement>(SelectorsMap.order.returnFormMainCheckbox);
```

And then you'll probably need to check if it exists later in the code